sources/host-ctr/cmd/host-ctr/main.go

```diff
@@ -834,6 +834,12 @@ func withPrivilegedMounts() oci.SpecOpts {
 			Source:      "/mnt",
 			Type:        "bind",
 		},
+		{
```

Comment:
nit: typo in commit message `s/host-containers now has/host-containers now have`